snmp/assets/dashboards/datadog_ndm_environment.json

```diff
@@ -1539,12 +1558,165 @@
                             "x": 6,
                             "y": 0
                         }
+                    },
+                    {
+                        "definition": {
+                            "legend_columns": [
+                                "avg",
+                                "min",
+                                "max",
+                                "value",
+                                "sum"
+                            ],
+                            "legend_layout": "auto",
+                            "requests": [
+                                {
+                                    "display_type": "bars",
+                                    "formulas": [
+                                        {
+                                            "formula": "query1"
+                                        }
+                                    ],
+                                    "queries": [
+                                        {
+                                            "data_source": "metrics",
+                                            "name": "query1",
+                                            "query": "sum:datadog.snmp_traps.invalid_packet{$SNMP_Device} by {snmp_device,reason}.as_count()"
```

Comment:
Should we also add `$Custom_tag` and `$Agent_hostanem`? Similar question for all queries

Reply:
Of course good catch thank you 🙇‍♂️